packages/amplication-code-gen-types/src/schemas/file.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$id": "https://amplication.com/schema/entityfield/properties/file.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "file",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "maxFileSize": {
+      "type": "number",
+      "minimum": 0
+    },
+    "allowedMimeTypes": {
+      "type": "array",
+      "additionalItems": true,
+      "minItems": 0,
+      "items": {
+        "$comment": "String that the MIME type of the file must contain or a option from the enum",
+        "anyOf": [
+          {
+            "type": "string"
+          },
+          {
+            "enum": ["image/", "audio/", "video/", "application/", "document/"]
```

Comment:
As previously discussed, we should either leave the `allowedMimeTypes` open or add all possible options currently registered as IANA.
The following types are missing:
- `font/`
- `model/`
- `example/`
- `text/`

Reply:
It is open (you can add any arbitrary string), the enum is just a way to make it easy for the users to add the common mime types, meanwhile, I will add these as well.